webapp/components/CommentForm/CommentForm.vue

```diff
@@ -1,28 +1,22 @@
 <template>
-  <ds-form v-model="form" @submit="handleSubmit">
+  <ds-form v-model="form" @submit="handleSubmit" class="comment-form">
     <template slot-scope="{ errors }">
       <ds-card>
         <!-- with client-only the content is not shown -->
         <hc-editor ref="editor" :users="users" :value="form.content" @input="updateEditorContent" />
-        <ds-space />
-        <ds-flex :gutter="{ base: 'small', md: 'small', sm: 'x-large', xs: 'x-large' }">
-          <ds-flex-item :width="{ base: '0%', md: '50%', sm: '0%', xs: '0%' }" />
-          <ds-flex-item :width="{ base: '40%', md: '20%', sm: '30%', xs: '30%' }">
-            <ds-button
-              :disabled="disabled && !update"
-              ghost
-              class="cancelBtn"
-              @click.prevent="handleCancel"
-            >
-              {{ $t('actions.cancel') }}
-            </ds-button>
-          </ds-flex-item>
-          <ds-flex-item :width="{ base: '40%', md: '20%', sm: '40%', xs: '40%' }">
-            <ds-button type="submit" :loading="loading" :disabled="disabled || errors" primary>
-              {{ $t('post.comment.submit') }}
-            </ds-button>
-          </ds-flex-item>
-        </ds-flex>
+        <div class="buttons">
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-14T17:12:59Z" title="Tuesday, January 14th 2020, 6:12:59 pm +01:00">Jan 14, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,28 +1,22 @@
 <template>
-  <ds-form v-model="form" @submit="handleSubmit">
+  <ds-form v-model="form" @submit="handleSubmit" class="comment-form">
     <template slot-scope="{ errors }">
       <ds-card>
         <!-- with client-only the content is not shown -->
         <hc-editor ref="editor" :users="users" :value="form.content" @input="updateEditorContent" />
-        <ds-space />
-        <ds-flex :gutter="{ base: 'small', md: 'small', sm: 'x-large', xs: 'x-large' }">
-          <ds-flex-item :width="{ base: '0%', md: '50%', sm: '0%', xs: '0%' }" />
-          <ds-flex-item :width="{ base: '40%', md: '20%', sm: '30%', xs: '30%' }">
-            <ds-button
-              :disabled="disabled && !update"
-              ghost
-              class="cancelBtn"
-              @click.prevent="handleCancel"
-            >
-              {{ $t('actions.cancel') }}
-            </ds-button>
-          </ds-flex-item>
-          <ds-flex-item :width="{ base: '40%', md: '20%', sm: '40%', xs: '40%' }">
-            <ds-button type="submit" :loading="loading" :disabled="disabled || errors" primary>
-              {{ $t('post.comment.submit') }}
-            </ds-button>
-          </ds-flex-item>
-        </ds-flex>
+        <div class="buttons">
```


I would typically suggest something a bit more descriptive, but I really like how you target this class as a grandchild of `comment-form` to apply `CSS`, so it doesn't make me nervous that the styling will be accidentally applied elsewhere.
Or that we be tempted to use `scoped`

Reply:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2020-01-15T07:40:52Z" title="Wednesday, January 15th 2020, 8:40:52 am +01:00">Jan 15, 2020</time>_

---


I know, it's counter-intuitive at first! Also my first time to use it like this but if we really follow the `rscss` suggestions and `VueJS Styleguide` all our components will have two-word-names and we'd only have these shorter and more generic class names as children of more specific classes. And I really like the readability of `buttons` in `comment-form` as opposed to BEM-style `comment-form__buttons`... :wink: